### DISCUSSION TITLE
Removal of the engine blast when wet

### DISCUSSION THREAD

**Discussion by PilotElias** [magknight/787aviatorsTracker#588] [Completed things] [answered]
It very much annoys me and is unrealistic. Would be nice if it got removed :-)
![X-Plane 2021-05-01 10-26-23](https://user-images.githubusercontent.com/82316636/116776289-da8d9f80-aa67-11eb-85a1-4fe8813a955d.png)

**Comment by KBNOtaku** [comment]
this is an XP11 default issue!
you can improve this by changing the textures for it!
its the "smoke_puff.png" file located at "\Resources\bitmaps\world\clouds"
If you want you can try and use the one I am using:
![smoke_puff](https://user-images.githubusercontent.com/56799893/118656409-c9c08600-b7ea-11eb-9b7a-5c5497285b79.png)

**Comment by aeroniemi** [comment, marked as answer]
The rain effect was disabled completely in v1.7.6
